src/registrar/tests/test_admin.py

```diff
@@ -1,13 +1,24 @@
 from django.test import TestCase, RequestFactory, Client
 from django.contrib.admin.sites import AdminSite
-from registrar.admin import DomainApplicationAdmin, ListHeaderAdmin, MyUserAdmin
+from registrar.admin import DomainApplicationAdmin, ListHeaderAdmin
```

Comment:
```suggestion
from registrar.admin import DomainApplicationAdmin, ListHeaderAdmin, MyUserAdmin, AuditedAdmin
```

remove the duplicate import below